packages/frontpage/app/(app)/post/[rkey]/_actions.tsx

```diff
@@ -5,8 +5,17 @@ import { createComment, deletePost, ensureUser, getPost } from "@/lib/data";
 export async function createCommentAction(formData: FormData) {
   const content = formData.get("comment") as string;
   const subjectRkey = formData.get("subjectRkey") as string;
+  const post = await getPost(subjectRkey);
+  if (!post) {
+    throw new Error("Post not found");
+  }
 
-  await createComment({ content, subjectRkey });
+  await createComment({
```

Comment:
if this is hard coded for post then the method should probably be renamed to reflect it is just for top level comments